File: primes.js
function Primes(maxNum) {
    var primes = [], nonprimes = [], n, i;
    for (n = 2; n <= maxNum; n++) {
        if (!nonprimes[n]) {primes.push(n)}
        for (i = n * 2; i <= maxNum; i += n) {
            nonprimes[i] = true;
        }
    }
    return primes;
}

console.time("PrimesTiming");
console.log(Primes(process.argv[2]));
console.timeEnd("PrimesTiming");

File: primes.py
import time

def prime_nums(max_num):
    primes = []
    nonprimes = set()

    for n in range(2, max_num + 1):
        if n not in nonprimes:
            primes.append(n)
        for i in range(n * 2, max_num + 1, n):
            nonprimes.add(i)
    return primes
    
number = int(input("Enter a number: "))
start = time.time()
print(prime_nums(number))
print(f'{time.time() - start} seconds')